osf/models/notable_domain.py

```diff
@@ -14,6 +14,7 @@ class Note(IntEnum):
         ASSUME_HAM_UNTIL_REPORTED = 1
         UNKNOWN = 2
         IGNORED = 3
+        UNVERIFIED = 4  # Timedout couldn't determine
```

Comment:
Did you run `python3 manage.py makemigrations`? I would expect a schema change in `note.choices` to generate an automigration, even if it's a functional no-op for the database.